src/index.html

```diff
@@ -1,13 +1,70 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="Html">
```

Comment:
```suggestion
<html lang="en">
```
Не порібно задавати клас для html. 
Глобально стилі прописуються, використовуючи селектор по тегу html